.circleci/config.yml

```diff
@@ -68,6 +68,7 @@ jobs:
             source .circleci/setup_env.sh
             make develop
             mkdir test-results
+            python tools/verify_install.py
```

Comment:
How long does this run? I was thinking you could give it its own job. Not important if its fast.

Reply:
It should be fairly fast.